content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cloud service management comes of age with Istio v1.1 performance update
+description: An overview of Istio 1.1 performance.
+publishdate: 2019-03-18
+subtitle: An overview of Istio 1.1 performance improvements
+attribution: Surya V Duggirala (IBM), Mandar Jog (Google), Jose Nativio (IBM)
+keywords: [Kubernetes, Istio, Docker, Container, performance, service mesh]
+weight: 101
+---
+
+Hyper-scale, microservice-based cloud environments have been exciting to build but challenging to manage. Along came Kubernetes (container orchestration) in 2014, followed by Istio (container service management) in 2017. Both open-source projects enable developers to scale container-based applications without spending too much time on administration tasks. 
+
+Now, new enhancements in Istio v1.1 deliver scale-up with improved application performance and service management efficiency.
+Simulations using our sample commercial airline reservation application show the following improvements, compared to Istio v1.0.
+
+**Application performance:**
```

Comment:
Please don't use ** to create titles. Use ## annotations instead as those will show up in the TOC and will look consistent.